authentication/forgot_password/forgot_password.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Forgot Password</title>
+
+    <link
+      rel="stylesheet"
+      href="https://clean-u-i.netlify.app/component/component.css"
+    />
+    <link rel="stylesheet" href="/common.css" />
+    <link rel="stylesheet" href="/authentication/authentication_common.css" />
+    <link rel="stylesheet" href="forgot_password.css" />
+  </head>
+  <body>
+    <div class="forgot-password-container">
+      <div class="authentication-details-container flex-row">
+        <div class="authentication-image-container">
+          <img src="/images/forgot_password.svg" alt="forgot-password-img" />
+        </div>
+        <div class="form-components-container flex-col">
+          <div class="wrapper p-2 my-2 flex-col">
+            <h2 class="my-2 form-heading text-sm">Forgot Password</h2>
```

Comment:
p2 my-2? can use more clear naming in future

Reply:
this convention  is followed in tailwind CSS and bootstrap too . So taken reference from it.
p-2 stands for padding in all directions of 1 rem and my-2 stands for margin along y-axis of 1 rem.